.github/workflows/default.yaml

```
@@ -1,12 +1,13 @@
-# Test the default branch, supported release branches, and their PRs
 name: GitHub CI
 
 on:
   push:
-    branches: [ "master", "v5", "auto" ]
+    # test this branch and staged PRs based on this branch code
+    branches: [ "master", "auto" ]
```

For v6, "this branch" in the comment describing this configuration option is "v6". This master change should be backported to v6 as

```suggestion
    branches: [ "v6", "auto" ]
```

or, if v6 maintainers do not want GitHub Actions to test v6 branch:

```suggestion
    branches: [ "auto" ]
```